AutoRip.js

```diff
@@ -375,20 +376,32 @@ function getCommandData() {
 }
 
 function processArray(array, fn, outputPath) {
-  var results = [];
-  return array.reduce((p, item) => {
-    return p.then(() => {
-      return fn(item, outputPath).then((data) => {
-        results.push(data);
-        return results;
-      });
+  var promises = [];

```

Comment:
This is the change that allows concurrent ripping. The way it's setup now, It needs to complete the full batch (return all promises) before starting a new one. Left the old function. Could make this optional at the start menu or in the config.

Reply:
Oh ok gotcha, yea I think that's fine for the time being. Solid state storage is much more common now than it was 6 years ago so I think the demand for sequential ripping would be somewhat low.